.gitlab-ci.yml

```diff
@@ -20,7 +24,7 @@ default:
 
 dev-pypi:
   tags:
-    - kube
+    - kube-new
```

Comment:
should be `kube-small-new`

Reply:
why? kube-new works, and I assumed these were kube and not kube-small for a reason.

Reply:
https://gitlab.polyswarm.io/externalci/polyswarm-cli/-/pipelines/34962